olp-cpp-sdk-core/src/http/Network.cpp

```diff
@@ -52,6 +52,14 @@ std::shared_ptr<Network> CreateDefaultNetworkImpl(size_t max_requests_count) {
 }
 }  // namespace
 
+void Network::SetDefaultHeaders(Headers /*headers*/) {}
```

Comment:
```suggestion
void Network::SetDefaultHeaders(Headers) {}
```

Reply:
a variable name is placed for consistency